src/Infrastructure/Adapter/LaneSectionMaker.php

```diff
@@ -30,7 +32,7 @@ private function resolvePoint(string $roadName, string $cityCode, ?string $house
     }
 
     /**
-     * @throws GeocodingFailureException
+     * @throws GeocodingFailureException|RoadGeocodingFailureException
```

Comment:
LaneSectionMaker ne peut pas déclencher une RoadGeocodingFailureException

```suggestion
     * @throws GeocodingFailureException
```